Makefile

```diff
@@ -1,2 +1,2 @@
 install:
-	cargo install --path .
+	cargo install --force --path rt-cli
```

Comment:
I added `--force` because I was unable to install using `cargo install --path` the first time. I think this is because i installed using `--git` first which conflicted. I will remove.

Reply:
thanks